### PR TITLE
Bad code

### PR DESCRIPTION
**Pull request by dan-hart**
This is bad code

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,4 +6,7 @@ Spring Boot
 Thymeleaf
 Java
 MySQL
-Bootstrap
\ No newline at end of file
+Bootstrap
+
+
+a;skdj;lfkahsldkfjh;lakjdl;fkjasl;dkfj;alskdjf
```
Comment (WillPohlman) [inline README.md]: Where you drunk dan? This is bad code

**Review by WillPohlman — CHANGES_REQUESTED** [review]
Remove bad code

File: README.md (hunk 1 above)
Comment (dan-hart) [reply README.md]: No, it was my cat. I fixed it

**Comment by WillPohlman** [comment]
Bad code was fixed.

**Review by WillPohlman — APPROVED** [review]
Bad code was removed

**Pull request merged**